make fails because build/bin directory doesn't exist

#### Expected behaviour

Compile shisui.

#### Actual behaviour

```
$ make shisui
[...]
cp main ./build/bin/shisui                                                                                                                                                                                                                                                              
cp: cannot create regular file './build/bin/shisui': No such file or directory                                                                                                                                                                                                          
make: *** [Makefile:14: shisui] Error 1                                     
```

#### Steps to reproduce the behaviour

Clone the repository
Install go
Run: make shisui

--- a/Makefile
+++ b/Makefile
@@ -11,6 +11,7 @@ GORUN = go run
 #? shisui: Build shisui
 shisui:
 	go build ./cmd/shisui/main.go
+	mkdir -p $(GOBIN)
 	cp main $(GOBIN)/shisui
 	@echo "Done building."
 	@echo "Run \"$(GOBIN)/shisui\" to launch shisui."

